From 77d1eb96913331b8f8a9c77debc60a0b2d63fe82 Mon Sep 17 00:00:00 2001
From: Wincent Colaiuta <win@wincent.com>
Date: Fri, 3 Sep 2010 01:23:15 +0200
Subject: [PATCH] Fix misleading advice to add 'memcache' to Gemfile

Commit 57144388f removed the hard-coded dependency on the memcache-client
gem, and added this warning advising people to install it if needed. The
problem is, however, that if people follow the advice literally and install
the 'memcache' gem, they will wind up with a completely different thing,
which is not API compatible with the memcache-client gem and which Rails
can't work with.

So, be explicit and tell users to install the 'memcache-client' gem.
---
 .../lib/active_support/cache/mem_cache_store.rb    |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/activesupport/lib/active_support/cache/mem_cache_store.rb b/activesupport/lib/active_support/cache/mem_cache_store.rb
index f32b562..9afe1b3 100644
--- a/activesupport/lib/active_support/cache/mem_cache_store.rb
+++ b/activesupport/lib/active_support/cache/mem_cache_store.rb
@@ -1,7 +1,7 @@
 begin
   require 'memcache'
 rescue LoadError => e
-  $stderr.puts "You don't have memcache installed in your application. Please add it to your Gemfile and run bundle install"
+  $stderr.puts "You don't have memcache-client installed in your application. Please add it to your Gemfile and run bundle install"
   raise e
 end
 require 'digest/md5'
-- 
1.7.2.2

